apps/andi/assets/css/app.scss

```diff
@@ -48,7 +48,7 @@ body {
 
 .error-msg {
   color: red;
-  display: block;
+  display: block !important;
```

Comment:
wait, what?! is there a reason we have to do this for css we control? Or is this overriding a 3rd party component?

Reply:
it was more just a temporary fix but we will change this back